api/v2/datadog/model_logs_archive_definition.go

```diff
@@ -39,8 +39,6 @@ func NewLogsArchiveDefinition(type_ string) *LogsArchiveDefinition {
 // but it doesn't guarantee that properties required by API are set
 func NewLogsArchiveDefinitionWithDefaults() *LogsArchiveDefinition {
 	this := LogsArchiveDefinition{}
-	var type_ string = "archives"
```

Comment:
That change looks weird? It looks like it only applies on that model.

Reply:
It is because the property is `readOnly`.